build.gradle

```diff
@@ -43,7 +48,7 @@ ext {
     okioVersion = '3.2.0'
     coilVersion = '2.2.2'
 
-    checkout3dsSdkVersion = '3.0.0'
+    checkout3dsSdkVersion = '3.0.1'
```

Comment:
Not sure if version hardcoded to `3.0.1` was expected,  shouldn't it be `file('version.resolved').getText()` ?

Reply:
It's a version of Checkout 3DS SDK, not our integration:
https://github.com/checkout/checkout-3ds-sdk-android/packages/929646

Reply:
aaa, I see